ServerCore/Pages/Teams/Index.cshtml.cs

```diff
@@ -1,13 +1,15 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.EntityFrameworkCore;
 using ServerCore.DataModel;
 using ServerCore.ModelBases;
 
 namespace ServerCore.Pages.Teams
 {
+    [Authorize(Policy = "IsEventAdminOrEventAuthor")]
```

Comment:
Since this is becoming an admin view, would it make sense to rename it and make the index be the player page?

Reply:
I think that causes a lot of issues with the navigation we have in place (getting to / from teams list) - for now if it's ok I'll leave it